docs/development/index.md

```diff
@@ -12,4 +12,38 @@ Fluidd is built using VueJS, and the Vuetify Framework to provide a cohesive,
 easily to implement UI.
 {: .fs-6 .fw-300 }
 
-Further details on development environments will come at a later date.
+## Running Fluidd locally
+
+### Install Node v14
+
+Follow the instructions from [NodeSource](https://github.com/nodesource/distributions/blob/master/README.md#deb)
+under "Node.js v14.x". For example, on Ubuntu flavored Linux distributions,
+run:
+
+```bash
+# Using Ubuntu
+$ curl -fsSL https://deb.nodesource.com/setup_14.x | sudo -E bash -
+$ sudo apt-get install -y nodejs
+```
+
```

Comment:
I would add a note to check that nodejs was installed properly by doing a ```nodejs --version``` and a ```npm --version```

Reply:
That's a very good point and always a nice thing to check!

Reply:
@matthewlloyd Would you mind adding it into your changes?